package/yast2-network.changes

```diff
@@ -2,9 +2,76 @@
 Mon Jul 10 07:06:17 UTC 2017 - gsouza@suse.com
 
 - Added support to storage-ng so it can verify if a disk is in
-network.
+  network.
 - Updated dependencies to include storage-ng.
-- 3.3.1
+
+-------------------------------------------------------------------
```

Comment:
Wow, wow! Now I see a lot of duplication in the changelog. According to this diff you are re-adding a lot of entries that are already below.